pkgs/development/libraries/libtickit/001-skip-test-18term-builder-on-macos.patch

```diff
@@ -1,27 +1,15 @@
-From 6179359c0b9247ae981b8b2a2897eabc921147fd Mon Sep 17 00:00:00 2001
```

Comment:
The header makes this a complete patch, otherwise it is just a diff. If we have it already we might as well keep it.

Reply:
You marked this as resolved but it isn't actually

Reply:
Sorry, @SuperSandro2000. There was a mixup while force pushing. Now its a proper patch.

Reply:
Is there anything else needed for it to be merged?